RIGS/templates/RIGS/event_detail_buttons.html

```diff
@@ -0,0 +1,33 @@
+<div class="btn-group btn-page">
+    <a href="{% url 'event_update' event.pk %}" class="btn btn-default"><span
+            class="glyphicon glyphicon-edit"></span> <span
+            class="hidden-xs">Edit</span></a>
+    {% if event.is_rig %}
+        <a href="{% url 'event_print' event.pk %}" target="_blank" class="btn btn-default"><span
+                class="glyphicon glyphicon-print"></span> <span
+                class="hidden-xs">Print</span></a>
+    {% endif %}
+    <a href="{% url 'event_duplicate' event.pk %}" class="btn btn-default" title="Duplicate Rig"><span
+            class="glyphicon glyphicon-duplicate"></span> <span
+            class="hidden-xs">Duplicate</span></a>
+    {% if event.is_rig %}
+        <a class="btn btn-default item-add modal-href event-authorise-request"
+                href="{% url 'event_authorise_request' object.pk %}">
+            <span class="glyphicon glyphicon-send"></span>
+            Authorisation Request
```

Comment:
An idea I just had: how about just adding "Last sent to", "Last sent at" and "last_sent_by" fields to the EventAuthorisation model. Update those whenever a teccie presses "send", and the history will be kept by reversion